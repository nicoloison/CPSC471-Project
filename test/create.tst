curl -F "name=ribs" -F "author_name=adjuric" -F "instructions=grill em" -F "prep_time=75" -F "portions=4" -F "description=saucy ribs" -F "image=@/home/liam/src/school/cpsc471/CPSC471-Project/test/Baby-Back-Ribs.jpg" localhost/recipes/create.php
curl localhost/recipes/create.php
curl -F "name=ribs" localhost/recipes/create.php
curl -F "name=ribs" -F "author_name=adjuric" localhost/recipes/create.php
curl -F "name=ribs" -F "author_name=adjuric" -F "instructions=grill em" localhost/recipes/create.php
curl -F "name=ribs" -F "author_name=adjuric" -F "instructions=grill em" -F "prep_time=75" localhost/recipes/create.php
curl -F "name=ribs" -F "author_name=adjuric" -F "instructions=grill em" -F "prep_time=75" -F "portions=4" localhost/recipes/create.php
curl -F "name=ribs" -F "author_name=adjuric" -F "instructions=grill em" -F "prep_time=75" -F "portions=4" -F "description=saucy ribs" -F "image=@/home/liam/src/school/cpsc471/CPSC471-Project/test/Baby-Back-Ribs.jpg" localhost/recipes/create.php
curl -F "name=ribs" -F "author_name=adjuric" -F "instructions=grill em" -F "prep_time=75" -F "portions=4" -F "description=saucy ribs" localhost/recipes/create.php
