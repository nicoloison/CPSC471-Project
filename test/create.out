{"result":"success","error":""}{"result":"failure","error":"missing parameter name"}{"result":"failure","error":"missing parameter author_name"}{"result":"failure","error":"missing parameter instructions"}{"result":"failure","error":"missing parameter prep_time"}{"result":"failure","error":"missing parameter portions"}{"result":"failure","error":"missing parameter description"}{"result":"failure","error":"database insertion error 1062 Duplicate entry 'ribs-adjuric' for key 'PRIMARY'"}{"result":"failure","error":"database insertion error 1062 Duplicate entry 'ribs-adjuric' for key 'PRIMARY'"}